app/assets/stylesheets/custom.css.scss

```diff
@@ -45,11 +44,11 @@ h1 {
 
 
 .nav.navbar-nav.navbar-right li a {
-    color: black;
+    color: #777;
```

Comment:
Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces